src/modm/platform/timer/stm32/advanced.hpp.in

```diff
@@ -368,10 +368,11 @@ public:
 
 	template<typename Signal>
 	static void
-	configureOutputChannel(OutputCompareMode mode, Value compareValue)
+	configureOutputChannel(OutputCompareMode mode, Value compareValue,
+			Value compareValue, PinState out = PinState::Enable)
```

Comment:
You have accidentally duplicated the parameter `Value compareValue`.

Reply:
*argh* a by-product of moving commits back and forth with patches ... sorry for any inconvenience caused.